docs/metrics/cohen-kappa.md

```diff
@@ -0,0 +1,114 @@
+# Cohen's Kappa
+
+Cohen's Kappa is a statistical metric that evaluates the reliability of an algorithm's predictions with similar goals
+as [accuracy](./accuracy.md) or [F<sub>1</sub>-score](./f1-score.md). However, the Cohen Kappa score magnifies the
+advantage a classifier has over a random classifier based on class frequencies. The Kappa statistic is a robust metric,
+particularly when there is significant class imbalance for binary and multiclass classification problems, which
+extends to more complicated ML tasks such as object detection.
```

Comment:
It would be nice to add an expected range of the metric along with a quick guidance on how to interpret the score.